fs/dtype_test.go

```diff
@@ -0,0 +1,18 @@
+package fs
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+)
+
+func TestRequiresRootNOP(t *testing.T) {
+
+	// This is a dummy test case that exist to call
```

Comment:
i presume you mean github.com/containerd/containerd/client_test.go?  i haven't tried this, but afaik none of the files in github.com/containerd/containerd/fs import github.com/containerd/containerd, so i'm not sure that would work.